test/parallel/test-stream-pipe-error-unhandled.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { Readable, Writable } = require('stream');
+
+process.on('uncaughtException', common.mustCall((err) => {
+  assert.strictEqual(err.message, 'asd');
+}));
+
+const r = new Readable({
+  read () {
```

Comment:
```suggestion
  read() {
```

Fix for lint failure https://travis-ci.com/nodejs/node/jobs/240921145#L247